src/components/cardUnidades/cardUnidades.jsx

```diff
@@ -0,0 +1,16 @@
+import styles from './CardUnidades.module.css'
```

Comment:
Esta quebrando por que não bate com o nome de arquivo
```suggestion
import styles from './cardUnidades.module.css'
```